Added duplicated tests run hint. (#46)
rubocop/minitest-style-guide: README.adoc

Choose only one to use – avoid mixing both approaches within one project.

> == Subclasses might run the parent tests methods twice.

Reviewer: I think the section header should only mention the topic area and not be too specific, e.g.:
```suggestion
== Subclassing Test Cases
```

Minitest uses Ruby classes, if a Minitest class inherits from another class, it will also inherit its methods 
causing Minitest to run the parent's tests twice.

[source,ruby]
----
# example

class ParentTest < Minitest::Test
  def test_1
    #... Run twice
  end
end

class ChildTest < ParentTest
  def test_2
    #...
  end
end
----

> In some cases, we want them to run the tests twice, but if not, avoid using subclasses or use minitest/spec, which disables 
> class inheritance between tests classes. You can measure how many duplicated tests you have using [this script](https://github.com/ignacio-chiazzo/ruby-minitest-analyzer).

Reviewer: I think it's unlikely someone will switch to `minitest/spec` because of this. Also, I think it's probably not very common to _want_ to run the tests twice?

```suggestion
In rare cases, we may want to run the tests twice, but in general avoid subclassing tests cases.

Note: The `minitest/spec` alternative syntax disable inheritance between test classes and so does not have this behavior.
```

Author: Good point 👍

Choose only one to use – avoid mixing both approaches within one project.

== Subclassing Test Cases

Minitest uses Ruby classes, if a Minitest class inherits from another class, it will also inherit its methods 
causing Minitest to run the parent's tests twice.

[source,ruby]
----

> # example

Reviewer 2: Can you tweak it?

```suggestion
# bad
```

Author: I prefer not to say `bad` since it's still a valid but rare use case.

Reviewer: Maybe
`# bad (unless multiple runs are the intended behavior)`

Reviewer: (I find the `good`/`bad` terminology that the guides use to be not ideal, since they don't really account for nuance or context).

class ParentTest < Minitest::Test
  def test_1
    #... Run twice
  end
end

class ChildTest < ParentTest
  def test_2
    #...
  end
end
----

> In rare cases, we may want to run the tests twice, but in general avoid subclassing tests cases.

Reviewer: Sorry, had a typo in my earlier suggestion:

```suggestion
In rare cases, we may want to run the tests twice, but in general avoid subclassing test cases.
```
